### DISCUSSION TITLE
请问目前支持Rocky Linux9.4吗？

### DISCUSSION THREAD

**Discussion by wqinf** [1Panel-dev/1Panel#5848] [Q&A]
请问目前支持Rocky Linux9.4吗？最小化安装Rocky Linux9.4之后，安装的时候报错。
![image](https://github.com/user-attachments/assets/c72d99a0-01e9-453a-bc60-20b8b2c69ae9)

**Comment by wanghe-fit2cloud** [comment]
- 截图中的问题是因为服务器没有 `tar` 命令导致的，手动安装一下吧
- 需要使用 `其他` 类型的安装脚本，如图：
<img width="954" alt="image" src="https://github.com/user-attachments/assets/921a1f1b-ce8c-4b52-90ed-95ae7cab20af">

**Reply by wqinf** [reply]
> * 截图中的问题是因为服务器没有 `tar` 命令导致的，手动安装一下吧
> * 需要使用 `其他` 类型的安装脚本，如图：
> 
> <img alt="image" width="954" src="https://private-user-images.githubusercontent.com/46734380/349412960-921a1f1b-ce8c-4b52-90ed-95ae7cab20af.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjEyMDg2MjksIm5iZiI6MTcyMTIwODMyOSwicGF0aCI6Ii80NjczNDM4MC8zNDk0MTI5NjAtOTIxYTFmMWItY2U4Yy00YjUyLTkwZWQtOTVhZTdjYWIyMGFmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MTclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzE3VDA5MjUyOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWY5OGUzMTkxMTI1MDYwZTEwMzFhNjIzODIyMjJmYWZjNmJhYzg3NDk1MzgxNTk3YjI3NGEzZWRjNzNjZmYwYWEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.Gnh__PEN4ciz38jrdzuLdiXp1hk2Vr-LM-6tFyL7-PA">

问题已解决，谢谢。
